openupgrade_scripts/scripts/repair/16.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,26 @@
+---Models in module 'repair'---
+---Fields in module 'repair'---
+repair       / repair.fee               / price_total (float)           : is now stored
+repair       / repair.line              / price_total (float)           : is now stored
+# DONE: recomputed in post_migration
+
+repair       / repair.line              / lot_id (many2one)             : relation is now 'stock.lot' ('stock.production.lot') [nothing to do]
+repair       / repair.order             / lot_id (many2one)             : relation is now 'stock.lot' ('stock.production.lot') [nothing to do]
+# NOTHING TO DO: renaming done in stock
+
+repair       / repair.order             / location_id (many2one)        : now a function
+repair       / repair.order             / product_uom (many2one)        : now a function
+# NOTHING TO DO: field already existed (was a onchange)
+
+repair       / repair.order             / picking_id (many2one)         : NEW relation: stock.picking
+repair       / stock.picking            / repair_ids (one2many)         : NEW relation: repair.order
+# NOTHING TO DO: new relation
+
+repair       / stock.picking.type       / is_repairable (boolean)       : NEW hasdefault: compute
```

Comment:
Is the default a good value?

Reply:
Hi @pedrobaeza thanks for your review
I think it is, it is checking whether the type is not incoming and has other types pointing to it with for return.
Anyhow, this new field allows then to create repair from the incoming picking so I suppose it is a good thing to have it possible by default ?